services/traction/conftest.py

```diff
@@ -49,7 +49,7 @@ async def _override_get_db():
 def test_app(override_get_db: Callable) -> FastAPI:
     ##Disable Security before app is loaded
     # TODO will need to override again so we can test auth itself
-    s.ENDPOINT_SECURITY_ENABLED = False
+    s.ENDPOINT_SECURITY_ENABLED = True
```

Comment:
If we want to run the tests this way, we should remove the infrastructure. This env_var was a hack.

Reply:
Yes agree this was just a patch to test out the authentication.

Reply:
leaving for ian's PR.